### ISSUE TITLE
ignored-users

### ISSUE THREAD

**Issue by cquiroz** [singingwolfboy/create-linked-clubhouse-story#2]
Hi, thanks for this it is looking great
I'm testing the `ignored-users` option but I'm getting this error

```
 ##[warning]Unexpected input(s) 'ignored-users', valid inputs are ['github-token', 'clubhouse-token', 'project-name', 'opened-state-name', 'closed-state-name', 'merged-state-name', 'comment-template', 'user-map']
 Run singingwolfboy/create-linked-clubhouse-story@v1.3
   with:
       github-token: ***
           clubhouse-token: ***
	   project-name: Explore
	   opened-state-name: Started
           merged-state-name: Done
           closed-state-name: Abandoned
           ignored-users: scala-steward
           comment-template: Clubhouse story: {{{ story.app_url }}}
```

I looked at the code and it seems it takes `ignored-users` maybe v1.3 refers to an older version?
Thanks for this tool, seems very useful

**Comment by singingwolfboy** [comment]
I'm glad you've found this project helpful! I just released version 1.4, which should make this warning go away.

**Comment by cquiroz** [comment]
Thanks a lot, 1.4 seems to work well, cheers
